application/forms/Config/UserGroup/AddMemberForm.php

```diff
@@ -1,129 +1,52 @@
 <?php
-/* Icinga Web 2 | (c) 2015 Icinga Development Team | GPLv2+ */
+/* Icinga Web 2 | (c) 2022 Icinga Development Team | GPLv2+ */
 
 namespace Icinga\Forms\Config\UserGroup;
 
 use Exception;
-use Icinga\Data\Extensible;
-use Icinga\Data\Filter\Filter;
-use Icinga\Data\Selectable;
 use Icinga\Exception\NotFoundError;
-use Icinga\Web\Form;
 use Icinga\Web\Notification;
+use ipl\Web\Control\SimpleSearchField;
+use ipl\Web\FormDecorator\IcingaFormDecorator;
 
 /**
  * Form for adding one or more group members
  */
-class AddMemberForm extends Form
+class AddMemberForm extends SimpleSearchField
 {
-    /**
-     * The data source to fetch users from
-     *
-     * @var Selectable
-     */
-    protected $ds;
-
-    /**
-     * The user group backend to use
-     *
-     * @var Extensible
-     */
     protected $backend;
 
-    /**
-     * The group to add members for
-     *
-     * @var string
-     */
```

Comment:
What's wrong here?

Reply:
It was accidentally removed when I rewrote the entire class.